src/core/actions.py

```diff
@@ -0,0 +1,27 @@
+from inspect import getmembers, isfunction, ismodule
+
+from src.jbi import process as jbi_process
+
+module_dict = {jbi_process.ACTION_KEY: jbi_process.actions}
+
+
+def get_action_context_by_key(key):
+    if key not in module_dict:
+        assert False, "Unknown key requested"
```

Comment:
Do you think raising a `ValueError` here might be a better choice? From the docs:
> Raised when an operation or function receives an argument that has the right type but an inappropriate value

That description seems pretty apt for this situation.

Reply:
resolved in this commit:
https://github.com/mozilla/jira-bugzilla-integration/pull/1/commits/8219d4187d7cd8cde7dd014367fdf65aa6a9d98a